pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -323,6 +323,38 @@ func (k *Kubernetes) ConfigServicePorts(name string, service kobject.ServiceConf
 	return servicePorts
 }
 
+// ConfigTmpfs configure the tmpfs.
+func (k *Kubernetes) ConfigTmpfs(name string, service kobject.ServiceConfig) ([]api.VolumeMount, []api.Volume) {
+	volumeMounts := []api.VolumeMount{}
+	volumes := []api.Volume{}
+	var count int
+	var volumeName string
+	for _, volume := range service.TmpFs {
+
+		volumeName = fmt.Sprintf("%s-tmpfs%d", name, count)
+		count++
+
+		// create a new volume mount object and append to list
+		volmount := api.VolumeMount{
+			Name:      volumeName,
+			MountPath: volume,
+		}
+		volumeMounts = append(volumeMounts, volmount)
+
+		var volsource *api.VolumeSource
+		//create tmpfs specific empty volumes
+		volsource = k.ConfigEmptyVolumeSource("tmpfs")
+
+		// create a new volume object using the volsource and add to list
+		vol := api.Volume{
+			Name:         volumeName,
+			VolumeSource: *volsource,
```

Comment:
@surajnarwade what I mean is that it's suppose to be Go conventions for naming, so volSource instead of volsource